.github/workflows/integration.yml

```diff
@@ -12,16 +12,16 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        test-suite: [TestEthereumE2ESuite, TestFabricE2ESuite]
+        test-suite: [TestEthereumMultipartyE2ESuite, TestEthereumGatewayE2ESuite, TestEthereumMultipartyTokensRemoteNameE2ESuite, TestEthereumNamespaceE2ESuite, TestFabricGatewayE2ESuite, TestFabricMultipartyE2ESuite]
```

Comment:
Actually can you pull any reference to `TestEthereumNamespaceE2ESuite`? It's just a subset of the main multiparty suite, and I think it's likely broken at the moment, and also probably not needed going forward due to the way identities work now.